functions/index.js

```diff
@@ -1,156 +1,389 @@
 const AWS = require('aws-sdk');
 const docClient = new AWS.DynamoDB.DocumentClient();
 const table = process.env.DYNAMO_DB_TABLE;
+const allowedRequestParameters = JSON.parse(process.env.ALLOWED_REQUEST_PARAMETERS);
+
+exports.handler = async (event, context, callback) => {
+  if (event.path == "/getGame") {
+    await getGame(event);
+  }
+  
+  if (event.path == "/listGames") {
+    await listGames(event);
+  }
 
-exports.handler = async (event, context) => {
   if (event.path == "/createGame") {
-    try {
-      let data = await createGame(event);
-      return { body: data };
-    } catch (err) {
-      return { error: err };
-    }  
+    await createGame(event);
   }
   
   if (event.path == "/modifyGame") {
-    try {
-      let data = await modifyGame(event);
-      return { body: data };
-    } catch (err) {
-      return { error: err };
-    }  
+    await modifyGame(event);
   }
   
   if (event.path == "/deleteGame") {
-    try {
-      let data = await deleteGame(event);
-      return { body: data };
-    } catch (err) {
-      return { error: err };
-    }  
+    await deleteGame(event);
   }
-  
-  if (event.path == "/getGame") {
-    try {
-      let data = await getGame(event);
-      return { body: data };
-    } catch (err) {
-      return { error: err };
-    }  
+
+  //Assign data to the callback function and apply the appropriate Status Code
+  async function getGame(event) {
+    await returnGetData(event)
+    .then((data) => {
+      if (data.hasOwnProperty('code') && data.hasOwnProperty('message') && data.hasOwnProperty('statusCode')) {
```

Comment:
This if statement indicates that you aren't properly managing errors in our returnGetData function